bin/get-amy.py

```diff
@@ -38,15 +35,25 @@ def main():
     }
 
     # Save.
-    with open(output_file, 'w') as writer:
-        yaml.dump(config, writer, encoding='utf-8', allow_unicode=True)
+    try:
+        with open(output_file, 'w') as writer:
+            yaml.dump(config, writer, encoding='utf-8', allow_unicode=True)
+    except TypeError:
+        # Output_file isn't a string, so it's a file.
+        yaml.dump(config, output_file, encoding='utf-8', allow_unicode=True)
 
 
-def fetch_info(base_url, url):
+def fetch_info(base_url, url, tags=None):
     '''Download and save data.'''
 
     address = base_url + url
 
+    # Consider filtering by tags.
+    if tags:
+        query_params = [urllib.parse.urlencode({'tag': tag}) for tag in tags]
+        query = '&'.join(query_params)
```

Comment:
What's wrong with multiple tag=XYZ in one query string?  If the 
receiving end does the right thing, that seems better than requiring 
another level of parsing.